source/common/upstream/load_balancer_impl.cc

```diff
@@ -254,7 +258,8 @@ bool ZoneAwareLoadBalancerBase::earlyExitNonLocalityRouting() {
   }
 
   // Do not perform locality routing for small clusters.
-  uint64_t min_cluster_size = runtime_.snapshot().getInteger(RuntimeMinClusterSize, 6U);
+  uint64_t const min_cluster_size =
```

Comment:
sorry, nit: we generally write the const first (`const uint64_t...`). Same below.